generated/nidigital/nidigital/session.py

```diff
@@ -2415,6 +2415,28 @@ def fetch_capture_waveform(self, site_list, waveform_name, samples_to_read, time
 
         return waveforms
 
+    @ivi_synchronized
+    def get_site_pass_fail(self, site_list):
+        '''get_site_pass_fail
+
+        Returns dictionary where each key is the site number and the value is the pass or fail result
+
+        Args:
+            site_list (str):
+
+
+        Returns:
+            pass_fail ({ int: bool, int: bool, ... }): Dictionary where each key is the site number and the value is the pass or fail result
+
+        '''
+        result_list = self._get_site_pass_fail(site_list)
+
+        site_list = self.get_site_results_site_numbers(site_list, enums.SiteResult.PASS_FAIL)
```

Comment:
Should this method be private?
If so, please look into whether the enum should be public.

Reply:
> Should this method be private?

Yes, there's already an issue for it - #1298.

> If so, please look into whether the enum should be public.

No, it should not be public. I added a note in #1298 to ensure it's removed from public API.